igor.dzetcyna/first/task 1/hash.rb

```diff
@@ -0,0 +1,14 @@
+# Знайти кількість ключів, значення яких починається на “yes”
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#english-comments
Please write all comments in english